core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -322,19 +333,19 @@ public GraphHopper setGraphHopperLocation(String ghLocation) {
     }
 
     public String getDataReaderFile() {
-        return dataReaderFile;
+        return osmFile;
```

Comment:
Or should we rename it to `getOSMFile`? This is how `GraphHopperOSM` calls this method.